src/Spice86.Core/Emulator/InterruptHandlers/Dos/DosInt21Handler.cs

```
@@ -419,6 +486,10 @@ public class DosInt21Handler : InterruptHandler {
         }
     }
 
+    public void CloseFileUsingFcb() {
+        _state.AX = 0;
```

What is this doing? Just silently ignoring the call?